KitMonitor/Server/Models/Database/Entities/Indicator.cs

```diff
@@ -1,27 +1,26 @@
 ﻿using System.ComponentModel.DataAnnotations;
 using System.ComponentModel.DataAnnotations.Schema;
 
-namespace KitMonitor.Server.Models.Database.Entities
-{
-    public class Indicator
-    {
-        [Key]
-        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
-        public long Id { get; set; }
-        public long IdMonitorSystem { get; set; }
-        [Required]
-        public IndicatorTypeEnum Type { get; set; }
+namespace KitMonitor.Server.Models.Database.Entities;
 
-        [ForeignKey("IdMonitorSystem")]
-        public MonitorSystem MonitorSystem { get; set; } = null!;
-        public IEnumerable<IndicatorLog> Log { get; set; } = new List<IndicatorLog>();
-    }
+public class Indicator
+{
+	[Key]
+	[DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+	public long Id { get; set; }
+	public long IdMonitorSystem { get; set; }
+	[Required]
+	public IndicatorTypeEnum Type { get; set; }
 
-    public enum IndicatorTypeEnum
-    {
-        Temperature,
-        Ph,
-        Tds,
-        WaterFlow
-    }
+	[ForeignKey("IdMonitorSystem")]
+	public MonitorSystem MonitorSystem { get; set; } = null!;
+	public IEnumerable<IndicatorLog> Log { get; set; } = new List<IndicatorLog>();
 }
+
+public enum IndicatorTypeEnum
```

Comment:
for transfers, you need to add numbers:
       Temperature = 1,
	Ph = 2,
	Tds = 3,
	WaterFlow = 4